frontend/src/features/admin-form/settings/components/EmailNotificationsSection/StatusTrackerToggle.tsx

```diff
@@ -22,6 +23,23 @@ export const StatusTrackerToggle = (): JSX.Element => {
 
   const { mutateMrfStatusTracker } = useMutateFormSettings()
 
+  const ToggleDescription = () => {
+    return (
+      <Text textStyle="body-2" color="secondary.400">
+        {t(
+          'features.adminForm.settings.emailNotifications.section.regular.statusTrackerDescription',
+        )}{' '}
+        {/* TODO: update with status tracking preview link here */}
```

Comment:
question: shall this todo be removed since the preview link is alr updated here?